go/felix/rules/rule_defs.go

```diff
@@ -101,6 +105,9 @@ type RuleRenderer interface {
 	ProtoRuleToIptablesRules(pRule *proto.Rule, ipVersion uint8) []iptables.Rule
 
 	NATOutgoingChain(active bool, ipVersion uint8) *iptables.Chain
+
+	DNATsToIptablesChains(dnats map[string]string) []*iptables.Chain
```

Comment:
Any way we can give `dnats` a better name (especially since it's the other way around to snats)? `extIPtoIntIP` maybe?